app/export.go

```diff
@@ -161,7 +158,12 @@ func (app *App) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs []str
 	counter := int16(0)
 
 	for ; iter.Valid(); iter.Next() {
-		addr := sdk.ValAddress(iter.Key()[1:])
+		key := iter.Key()
+		if len(key) <= 2 {
+			app.Logger().Error("unexpected key in staking store", "key", key)
+			continue
+		}
+		addr := sdk.ValAddress(key[2:])
```

Comment:
Why this is changed from `[1:]` to `[2:]`?

Reply:
I think [1:] was a default generated value 
[2:] works for for validator keys 

I don't think we have ever used this function before, it gets triggered if we use the  `--for-zero-height`  when exporting